internal/driver/glfw/window.go

```diff
@@ -435,16 +435,16 @@ func (w *window) Close() {
 	w.viewLock.Unlock()
 	w.viewport.SetShouldClose(true)
 
-	w.canvas.walkTrees(nil, func(node *renderCacheNode) {
-		switch co := node.obj.(type) {
+	w.canvas.WalkTrees(nil, func(node *common.RenderCacheNode) {
+		switch co := node.Obj().(type) {
```

Comment:
I think you'd be better off with a simple if here.

Reply:
Absolutely :) thank you. I wasn't sure if I should change it, because it seems like it expected to handle other kind of CanvasObjects 🤔

Reply:
Updated! :)